packages/console/src/components/Steps/components/StepItem/Status.module.scss

```diff
@@ -0,0 +1,31 @@
+.status {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 20px;
+  height: 20px;
+  border-radius: 10px;
+  background-color: var(--color-surface-variant);
+  color: var(--color-text-link);
+  font: var(--font-body-3);
+  // Specify font-weight manually since we don't have a font token to fit this case
+  font-weight: 600;
```

Comment:
Should add new font token lib cc: @charIeszhao  otherwise, it may break our current font convention.  Besides it needs to be aligned with the figma token.

Reply:
It's more of a design issue, I think. Should double check with designers

Reply:
This is a particular case, the text needs to be bold, and we don't have this font token in our design system.
Maybe we should not use the font token here.